src/helpers/loginSucces.js

```diff
@@ -0,0 +1,66 @@
+import Swal from 'sweetalert2';
+import { createUser, iniciarSesion } from './login';
+
+let obj = [];
+
+let visibilidad = {
+	user: true,
+	home: false,
+	idUser: 0,
+};
+
+export const loginSucces = (user, password) => {
+	const validador = iniciarSesion(user, password);
+
+	visibilidad.idUser = validador.idUser;
+
+	if (validador.estado === false) {
+		Swal.fire({
+			icon: 'success',
+			title: 'Bienvenido',
+		});
+		visibilidad.user = true;
+		visibilidad.home = false;
+	} else {
+		Swal.fire('Datos incorrectos', 'intente nuevamente', 'question');
+		visibilidad.user = false;
+		visibilidad.home = true;
+	}
+
+	const formulario = document.querySelector('#fnewUsuario');
```

Comment:
you shouldn't use this for change the state of a component
You are not using React, instead you are using native js, it's not the best practice